### PR TITLE
Added https://  mirrors for mva-overlay, raiagent, rainyday, rasdark,…

### PR DESCRIPTION
**Pull request by Massimo-B**
… unity, vaca; #542340

### CODE REVIEW

**Comment by mgorny** [comment]
Would that be a problem if I asked you to put https first?

**Comment by Massimo-B** [comment]
Why? The sorting before wasn't consistent either. Then most sorting was git:// first.
Anyway, the precedene is done my layman.cfg, for my case ignoring git:// completely (but this is still the fallback if there is only a git:// mirror specified:
```
protocol_filter : http, https
```

**Comment by mgorny** [comment]
Well, because https is kinda-secure and kinda-portable ;-). And yes, I know it wasn't like this before. Nevertheless, we enforce that for new entries and I'd happily accept a commit changing it for all existing entries. I just haven't gotten to do that myself.

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -1900,8 +1900,8 @@ FIN
     <owner>
       <email>bug-wranglers@gentoo.org</email>
     </owner>
-    <source type="rsync">rsync://rsync.gentoo.org/gentoo-portage</source>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: Not here, rsync goes first for gentoo.

**Comment by mgorny** [comment]
Merged both, in reverse order. 80ee7bf134936b3d6b3 and bc5ca93ede8a108e73f. Thanks!

**Pull request closed without merging**